site/content/en/docs/Installation/Terraform/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Deploy GKE/AKS cluster and install Agones using Terraform"
```

Comment:
I'd suggest dropping the GKE/AKS part here and just say Kubernetes instead. From the guides below it's clear which platforms are supported.